hack/vendor.sh

```diff
@@ -69,7 +69,13 @@ mv tmp-digest src/github.com/docker/distribution/digest
 mkdir -p src/github.com/docker/distribution/registry
 mv tmp-api src/github.com/docker/distribution/registry/api
 
+# This test file checks that resumable support is available in digest library.
+# This is not at all needed in the docker daemon at this time. We remove it to
+# avoid compile-time issues with tests.
+rm -f src/github.com/docker/distribution/digest/digester_resumable_test.go # dependencies not required.
```

Comment:
This prevents pulling in a dependency that is only needed by the registry server. Vendoring code should be able to trim  packages to remove unused code.